src/app/models/transmart-models/transmart-default-table-representation.ts

```diff
@@ -0,0 +1,4 @@
+export class TransmartDefaultTableRepresentation {
```

Comment:
isn't a bit redundant now that we have TransmartTableState and TransmartDefaultTableRepresentation? TransmartDefaultTableRepresentation is just a special case of TransmartTableState, right?